build.sbt

```diff
@@ -60,12 +60,12 @@ val buildDebSettings = Seq(
 
 val `membership-common` =
   (project in file("membership-common"))
-    .enablePlugins(DynamoDBLocalPlugin, SbtPgp, Sonatype)
+    .enablePlugins(SbtPgp, Sonatype)
```

Comment:
the dynamo tests were for promo code, which are not used by members-data-api, so I deleted the code and tests.